### PR TITLE
Archive notification

### PR DESCRIPTION
**Pull request by gaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,137 +1,3 @@
...
-* Dev group: http://groups.google.com/group/jclouds-dev
-* Twitter: http://twitter.com/jclouds
-
-
-License
--------
-Copyright (C) 2009-2012 jclouds, Inc.
-
-Licensed under the Apache License, Version 2.0
+jclouds moved to Apache in 2013 and archived this repository.  Please visit our
```
Comment (demobox) [inline README.md]: [minor] Add link(s) to the mailing list and/or new project site?

**Comment by demobox** [comment]
On a related note, should we also change the description of the repo from its current value:

> jclouds is an open source library that helps you get started in the cloud and reuse your java development skills. Our api allows you to freedom to use portable abstractions or cloud-specific features. We support many clouds including Amazon, VMWare, Azure, and Rackspace. http://jclouds.apache.org/

?

File: README.md (hunk 1 above)
Comment (gaul) [reply README.md]: Done.

**Comment by gaul** [comment]
> On a related note, should we also change the description of the repo from its current value:

I removed the description.

**Pull request merged**